lib/integrations/awsoidc/create_ec2ice.go

```diff
@@ -77,6 +91,9 @@ func (req *CreateEC2ICERequest) CheckAndSetDefaults() error {
 type CreateEC2ICEResponse struct {
 	// Name is the Endpoint ID.
 	Name string
+
+	// CreatedEndpoints contains the name of created endpoints and their Subnet.
+	CreatedEndpoints map[string]string
```

Comment:
Suggestion: return a list of endpoint instances instead of a map. Returning the instances is a stronger future-proof design, as we could easily return more information in the future. It's also better self-documented than a map of strings.

```suggestion
	// Endpoints are the created endpoints.
	// Contains the endpoint name and its subnet.
	Endpoints []*EC2ICEEndpoint
```

Reply:
Sounds good :+1: